src/Configuration/test/Config.FunctionalTests/Microsoft.Extensions.Configuration.FunctionalTests.csproj

```diff
@@ -5,6 +5,13 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\testassets\Test.Common\Microsoft.Extensions.Configuration.Test.Common.csproj" />
```

Comment:
Full path of this project is `src/Configuration/test/Config.Test.Common/Microsoft.Extensions.Configuration.Test.Common.csproj` So, this should be `..\Config.Test.Common/Microsoft.Extensions.Configuration.Test.Common.csproj`

Reply:
Ok i'll give that a shot